pandas/tests/generic/test_generic.py

```diff
@@ -970,3 +970,10 @@ def test_deprecated_get_dtype_counts(self):
         df = DataFrame([1])
         with tm.assert_produces_warning(FutureWarning):
             df.get_dtype_counts()
+
+    # test case where offset and index[0] overlap
```

Comment:
move this near other tests for first: look in pandas/tests/frame/test_timeseries and series....